examples/Peripherals/HardwareTimer/countermode_up_down_centered/countermode_up_down_centered.ino

```diff
@@ -0,0 +1,123 @@
+/*
+  Countermode up down centered
+  This example shows how to configure HardwareTimer's counter mode.
+  
+  The following countermodes are demonstrated:
+  LL_TIM_COUNTERMODE_UP
+   _ — ‾ | _ — ‾ | _ — ‾ | _ — ‾ |
+  
+  LL_TIM_COUNTERMODE_DOWN
+   ‾ — _ | ‾ — _ | ‾ — _ | ‾ — _ |
+  
+  LL_TIM_COUNTERMODE_CENTER_UP_DOWN (with over/undeflow callback)
+  _ — ‾ | ‾ — _ | _ — ‾ | ‾ — _ |
+
+  LL_TIM_COUNTERMODE_CENTER_UP_DOWN (with only undeflow callback)
+  _ — ‾ ‾ — _ | _ — ‾ ‾ — _ |
```

Comment:
This figure would be more accurate:
`_ — ‾ — | _ — ‾ — |`

Reply:
Not sure why you think the first channel interrupt (25%) won't print on the way down (it does).